Userland/Libraries/LibWeb/Bindings/WindowObject.cpp

```diff
@@ -650,6 +651,17 @@ JS_DEFINE_NATIVE_FUNCTION(WindowObject::screen_y_getter)
     return JS::Value(impl->screen_y());
 }
 
+JS_DEFINE_NATIVE_FUNCTION(WindowObject::origin_getter)
```

Comment:
Please add a spec link above the function: `https://html.spec.whatwg.org/multipage/webappapis.html#dom-origin`